context/log4j2/src/main/java/brave/context/log4j2/Log4j2Context.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2013-2020 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package brave.context.log4j2;
+
+import brave.propagation.CorrelationField;
+import org.apache.logging.log4j.ThreadContext;
+
+/** Integrates Log4j 2's {@link org.apache.logging.log4j.ThreadContext} */
+public final class Log4j2Context extends CorrelationField.Context {
```

Comment:
Since package name is `log4j2` think this can just be `Log4jContext`

Reply:
I was going by the marketing on their website and kinda prefer the match on package name..

Reply:
obviated as the type is hidden